source/GameWindow.cpp

```diff
@@ -130,8 +146,8 @@ bool GameWindow::Init()
 		flags |= SDL_WINDOW_MAXIMIZED;
 	
 	// The main window spawns visibly at this point.
-	mainWindow = SDL_CreateWindow("Endless Sky", SDL_WINDOWPOS_UNDEFINED, 
-		SDL_WINDOWPOS_UNDEFINED, windowWidth, windowHeight, flags);
+	mainWindow = SDL_CreateWindow("Endless Sky", gameWindow.x, 
+		gameWindow.y, gameWindow.w, gameWindow.h, flags);
```

Comment:
Does sending windowpos_undefined for x and y not render in the center of the screen when using a non-primary display?

Reply:
It does, but that assumes display 0.
For example if you have 2x1920x1080 resolution diplays side by side, the x coordinate of display 1 content starts at 1920 when full screen, and even higher if not full screen. hance the gameDisplay.x += rather then gameDisplay.x =
Same for y if you have displays stacked vertically.

Reply:
Seems like we can instead set `SDL_WINDOWPOS_CENTERED` and avoid the extra code. Or does that also assume display 0, or does that stretch across displays?

Reply:
hmm... I don't know, but I'll get back with an answer.

Reply:
Yup, that also only works for display 0.